opm/input/eclipse/EclipseState/Grid/EclipseGrid.hpp

```diff
@@ -234,23 +254,34 @@ namespace Opm {
 
         static bool hasEqualDVDEPTHZ(const Deck&);
         static bool allEqual(const std::vector<double> &v);
+        std::vector<EclipseGridLGR> lgr_children_cells;
```

Comment:
I wonder if the variable name lgr_children_cells might be a bit misleading—it sounds more like a vector of cells rather than grids. What do you think about renaming it to something like lgr_eclipse_grids to make it clearer?